prow/plugins/trigger/push.go

```diff
@@ -85,7 +85,7 @@ func handlePE(c Client, pe github.PushEvent) error {
 			labels[k] = v
 		}
 		labels[github.EventGUID] = pe.GUID
-		pj := pjutil.NewProwJob(pjutil.PostsubmitSpec(j, refs), labels, j.Annotations)
```

Comment:
do we still need this definition of a function? I guess if we want to avoid next misses, we should get rid of it.

Reply:
What do you mean by get rid of? Replacing it with a straight `prowapi.ProwJob{}`?

Reply:
I am thinking if making 4th param mandatory makes sense.

Reply:
it was explained that we had this discussion before and it was decided to follow-up like this